week 3 assignment

hcd00 opened the pull request.

Commit:
week 3 assignment

AnnaSolovykh commented on 02-express-tutorial/app.js:
```diff
@@ -1 +1,74 @@
+
+//app.post()
+
+app.all('./public', (req, res, next) => {
```
I noticed that the catch-all route for handling page not found conditions needs some adjustment. According to the assignment requirements:

> An app.all statement after these to handle page not found conditions

Your current implementation uses app.all('./public', ...) with next(), which doesn't quite match the intended functionality. For a proper catch-all route, you would typically use an asterisk to match any unhandled routes.

AnnaSolovykh reviewed (changes requested):
Your implementation shows a good understanding of Express.js and includes all the required API endpoints. The client-side JavaScript for fetching and displaying products is also well implemented.
There is only one issue that needs to be addressed. Please make this adjustment and then request a review of your updated assignment.

hcd00 committed:
Update app.js

Addressed app.all not properly handling not found pages. Now returns 404 HTTP status with message.

AnnaSolovykh reviewed (approved):
Well done, approved!
